modules/gitops-argo-cd-properties.adoc

```diff
@@ -59,26 +78,44 @@ a|* `<enabled>` - Toggle high availability support globally for Argo CD.
 
 |`image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
 
+|`Import`    |Import configuration options for Argo CD.| `__<object>__`
+a|* `<Name>` - The name of an `ArgoCDExport` resource from which data can be imported.
+  * `<Namespace>` - The namespace of `ArgoCDExport` resource is set to the same namespace as ArgoCD by default.
+
 |`ingress`    |Ingress configuration options.| `__<object>__` |
 
 |`initialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|`__<empty>__` |
 
+|`monitoring`    |Defines the workload status monitoring configuration status for your instance.| `__<object>__`
```

Comment:
```suggestion
|`monitoring`    |Defines the workload status monitoring configuration for your instance.| `__<object>__`
```
This did not seem correct to me with both "status" words there. Is the meaning still same with my suggestion?